packages/office-ui-fabric-react/src/components/Button/ButtonThemes.ts

```diff
@@ -93,17 +95,38 @@ export function primaryStyles(theme: ITheme): IButtonStyles {
   return {
     root: {
       backgroundColor: theme.palette.themePrimary,
-      color: theme.palette.white
+      color: theme.palette.white,
+      selectors: {
+        [MS_HIGHCONTRAST_ACTIVE]: {
+          color: 'Window',
+          backgroundColor: 'WindowText',
+          MsHighContrastAdjust: 'none'
```

Comment:
if we actually need this, it's really bad practice, we shouldn't be having usage-necessary images being background images. this is something we can definitely do by tweaking the HTML, or using a :before

Reply:
@phkuo, I'm not sure what the issue is here, there is no background image. The idea is just that primary buttons should use a background color instead of a border like normal buttons. Can you clarify what you'd like to see changed here?

Reply:
Isn't -ms-high-contrast-adjust:none used to show the background image? If yes, see previous comment.

If it's only used so that the background color shows, then shouldn't setting background color directly (as is already done here) be sufficient?

Reply:
`-ms-high-contrast-adjust` affects a few different things as well. Namely, it turns off the `Window` colored highlight that Edge (but not IE) gives all text. It's invisible most of the time, but any time there is an intentional background color or the text is close to other elements it covers things up.

Without setting it here, things like this happen.
![capture](https://user-images.githubusercontent.com/12915480/32921000-94db9ed4-cae0-11e7-89c8-cb7ffbbe75c7.PNG)